pyanaconda/startup_utils.py

```diff
@@ -656,7 +657,16 @@ def initialize_security():
     certificates_proxy = SECURITY.get_proxy(CERTIFICATES)
     import_task_path = certificates_proxy.ImportWithTask()
     task_proxy = SECURITY.get_proxy(import_task_path)
-    sync_run_task(task_proxy)
+    try:
+        sync_run_task(task_proxy)
+    except SecurityInstallationError as e:
+        log.error(e)
+        print(_("\nAn error occurred during certificate import:"
```

Comment:
This is so far ever going only via kickstart, right ? Then I would suggest adding that to the error message, something like:

`An error occurred during certificate import from kickstart:`

That should make it easier to track down & fix the issue, especially if its a third party seeing the error message and reporting it back (eq. an automated kickstart based deployment going wrong somewhere).